sheets/resize2fs

```diff
@@ -1,6 +1,6 @@
 # resize2fs
 
-# ext2/ext3/ext4 file system resizer
+# ext2/ext3/ext4 file system resize tool
```

Comment:
That doesn't sound natural to me. I'd say the original was fine, or: "file system resizing tool"